### PR TITLE
feat: Build '/solutions/sovereign-cloud' page and modal form

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Build '/solutions/sovereign-cloud' page based on [copydoc](https://docs.google.com/document/d/1XvdgVN87yqI9xOtFBaK9Z_Au8PTeHlQICzlYrgUKxuQ/edit) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions%2Fsovereign-cloud?node-id=7-1254&t=NWmCG5I6aETEhEtb-0)
- Build modal form based on [copydoc](https://docs.google.com/document/d/1GkR3cbZbe7-gbwADceV8CkZK1xWI55nx8rSmsCLyuio/edit)

## QA

- Go to https://canonical-com-1280.demos.haus/solutions/sovereign-cloud
- Check the content matches that in the [copydoc](https://docs.google.com/document/d/1XvdgVN87yqI9xOtFBaK9Z_Au8PTeHlQICzlYrgUKxuQ/edit)
- Check on all screen sizes
- Check the modal opens and that the content matches the [copydoc](https://docs.google.com/document/d/1GkR3cbZbe7-gbwADceV8CkZK1xWI55nx8rSmsCLyuio/edit)
- Check the modal forms submits properly

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12009

### CODE REVIEW

**Comment by petesfrench** [comment]
@Sophie-32 @lyubomir-popov Can you take a look please

**Comment by petesfrench** [comment]
@lyubomir-popov Can you take another look please?

**Comment by petesfrench** [comment]
@Sophie-32 Could you take a look at this please

**Review by britneywwc — APPROVED** [review]
Very nice, I like it. Left some minor suggestions but otherwise it's a big +1 from me

File: templates/solutions/sovereign-cloud/index.html [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+        <div class="p-section--shallow">
+          <h1>Control your data and infrastructure with a sovereign cloud</h1>
+        </div>
+        <div class="p-section--shallow">
+          <p>
+            In a world of growing compliance challenges and international-level security risks, a sovereign cloud helps organisations, governments and their institutions safeguard their data, access and infrastructure.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a href="/contact-use"
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/solutions/sovereign-cloud/index.html:27]: oops x3

**Comment by lyubomir-popov** [comment]
These look blurry, cdn issue again I guess?

**Comment by lyubomir-popov** [comment]
Thanks @petesfrench - the only issue I see is blurry logos, could you please double check the cdn versions and why are they blurry. I believe the original assets were exported at 3x the requireed resolution, sao there shouldn't be a problem
image

**Comment by petesfrench** [comment]
@lyubomir-popov I have update to not go through the CDN. It seems we are having some kind of issue either through the python image template or the CDN. I have made an issue to look into this here: https://warthogs.atlassian.net/browse/WD-12738

**Pull request merged**